src/tests/ftest/util/server_utils.py

```diff
@@ -245,49 +290,46 @@ def stop_server(setname=None, hosts=None):
                 SESSIONS[setname].kill()
             SESSIONS[setname].wait()
         print("<SERVER> server stopped")
+
     except Exception as error:
         print("<SERVER> Exception occurred: {0}".format(str(error)))
         raise ServerFailed("Server didn't stop!")
 
     if not hosts:
         return
 
-    # make sure they actually stopped
-    # but give them some time to stop first
+    # Make sure the servers actually stopped.  Give them time to stop first
+    # pgrep exit status:
+    #   0 - One or more processes matched the criteria.
+    #   1 - No processes matched.
+    #   2 - Syntax error in the command line.
+    #   3 - Fatal error: out of memory etc.
     time.sleep(5)
```

Comment:
Ditto for the `sleep()` here.

Reply:
Again I agree - never been a fan of sleeps (there is another on line 284).  The Popen.poll() and Popen.wait() calls above should handle ensuring the stop commands have completed.  I'm just weary of removing an undocumented piece of code that may handle a corner case that prevents CI failures - especially when it is slightly out of the scope of the PR.